### PR TITLE
New package: loudgain-0.6.8

### PR DESCRIPTION
**Pull request by motorto**
Asked on #23054

### CODE REVIEW

File: srcpkgs/loudgain/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'loudgain'
+pkgname=loudgain
+version=0.6.8
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="libebur128-devel ffmpeg-devel taglib-devel pkg-config"
```
Comment (Duncaen) [inline srcpkgs/loudgain/template:7]: `pkg-config` should only be in `hostmakedepends`

Comment (motorto) [reply srcpkgs/loudgain/template]: Thanks , don't know how I missed that , just fixed it !

**Comment by motorto** [comment]
Hey @duncaen could you re review it again and if it ready push it ?

File: srcpkgs/loudgain/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+makedepends="libebur128-devel ffmpeg-devel taglib-devel"
+short_desc="Versatile ReplayGain 2.0 loudness normalizer,EBU R128/ITU BS.1770"
+maintainer="André Cerqueira <acerqueira021@gmail.com>"
+license="BSD-2-Clause"
+homepage="https://github.com/Moonbase59/loudgain"
+distfiles="https://github.com/Moonbase59/loudgain/archive/v${version}.tar.gz"
+checksum=1137c193ad941b366e87c5d84ccc95a7aa8571affc060db0bd1cf72c489aeaee
+
+pre_build() {
+	mkdir -p build && cd build
```
Comment (Duncaen) [inline srcpkgs/loudgain/template:16]: This shouldn't be necessary, the cmake build_style already sets up the build directory at configure time before pre_build.
https://github.com/void-linux/void-packages/blob/master/common/build-style/cmake.sh#L6

**Review by Duncaen — CHANGES_REQUESTED** [review]

**Comment by motorto** [comment]
Thanks for the insight wasn't aware of that. @Duncaen

**Comment by the-maldridge** [comment]
Closed by b766fcd215

**Pull request closed without merging**